pyproject.toml

```diff
@@ -7,7 +7,7 @@ dependencies = [
     "attrs",
     "catkin-pkg==1.0.0 ; sys_platform == 'linux'",
     "click>=8.2",
-    "craft-application[remote]>=5.2.0,<6.0.0",
+    "craft-application[remote]>=5.3.0,<6.0.0",
```

Comment:
Did you update the lockfile?

Reply:
`make setup` won't do that since it's frozen, but we could do a `uv lock`

Reply:
I did a thing: https://github.com/canonical/starbase/pull/396